simulator/worlds/blue_frames.world

```diff
@@ -0,0 +1,248 @@
+<sdf version='1.6'>
```

Comment:
name of the file must be intutive, like this is world file then name something like basic_arena ... something something